internal/benchmarks/Makefile

```diff
@@ -0,0 +1,6 @@
+all: gen benchmark
+
+benchmark:
+	go test -v -run="^6" -bench=. ./...
```

Comment:
what is ^6?

Reply:
It's a random string that doesn't match any pattern so that benchmarks will run exclusively.

Reply:
I recall why I used it, the Makefile interpretes "$", and the usual pattern is `-run="^$"`